src/Microsoft.ML.Data/Model/Onnx/SaveOnnxCommand.cs

```diff
@@ -228,10 +266,68 @@ private void Run(IChannel ch)
 
             if (!string.IsNullOrWhiteSpace(Args.OutputModelFile))
```

Comment:
>OutputModelFile [](start = 48, length = 15)

Did you set the visibility on these? I don't see the `DataCommand.cs` in this list here.

Reply:
These are not exposed in the EP since I created the arguments class for EP input. 

---
In reply to: [191821856](https://github.com/dotnet/machinelearning/pull/248#discussion_r191821856) [](ancestors = 191821856)

Reply:
Sure, but I got the sense you didn't want to do that. It means a fair amount of duplication that I think is probably unnecessary.

---
In reply to: [191901052](https://github.com/dotnet/machinelearning/pull/248#discussion_r191901052) [](ancestors = 191901052,191821856)